back-end/package.json

```diff
@@ -13,7 +13,7 @@
     "update-db": "npm run prisma-generate && npm run prisma-migrate-dev",
     "docker-stop-containers": "npx run-script-os",
     "docker-stop-containers:windows": "@powershell -NoProfile -ExecutionPolicy Unrestricted -Command ./bin/DockerStopContainers/DockerStopContainers.ps1",
-    "docker-stop-containers:default": "docker stop $(docker ps -a -q)",
+    "docker-stop-containers:default": "docker stop $(docker ps -a -q) || echo 'Nothing to stop'",
```

Comment:
If the developer has no container running, this will have an error, so I gave it an exit